### PR TITLE
Improve horizon ingestion docs

### PR DESCRIPTION
**Pull request by jacekn**

### CODE REVIEW

File: content/docs/run-api-server/ingestion.mdx [hunk 1]
```diff
@@ -162,15 +162,15 @@ export RETRY_BACKOFF_SECONDS=20
 If Horizon was previously running, first ensure it is stopped. Then, run the following commands in parallel:
 
 1. `stellar-horizon db reingest range --parallel-workers=64 1 16999999`
-2. `stellar-horizon db reingest range --parallel-workers=24 17000000 <latest_ledger>`
+2. `stellar-horizon db reingest range --parallel-workers=20 17000000 <latest_ledger>`
 
 (Where you can find `<latest_ledger>` under [SDF Horizon's](https://horizon.stellar.org/) `core_latest_ledger` field.)
 
-When saturating an RDS instance with 15K IOPS capacity:
+When saturating a database instance with 15K IOPS capacity:
```
Comment (2opremio) [inline content/docs/run-api-server/ingestion.mdx:169]: I don't think it's bad to be specific about RDS since (AFAIU) the IOPS metrics are amazon specific?

Comment (jacekn) [reply content/docs/run-api-server/ingestion.mdx:169]: I changed another place where we mention IOPS to be cloud provider agnostic too, other than that the metric should be available in most cloud providers (or even on bare metal)

Comment (2opremio) [reply content/docs/run-api-server/ingestion.mdx:169]: Ah, I didn't know the term is applicable to every cloud provider in the same way. OK

**Review by 2opremio — APPROVED** [review]

File: content/docs/run-api-server/ingestion.mdx [hunk 2]
```diff
@@ -132,7 +132,7 @@ processes in parallel:
 
 This is due to first 17 million ledgers being almost empty whilst the rest are much more packed. Having a single Horizon instance with enough workers to saturate the IO capacity of the machine for the first 17 million would kill the machine when reingesting the rest (during which there is a higher CPU and memory consumption per worker).
 
-64 workers for (1) and 24 workers for (2) saturates the IO capacity of an RDS instance with 5TB of General Purpose SSD storage. Again, as the DB storage grows, a larger number of workers should be considered.
+64 workers for (1) and 20 workers for (2) saturates instance with 64GB or RAM and 15K IOPS. Again, as the DB storage grows, a larger number of workers and faster storage should be considered.
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:135]: nit: typo here
```suggestion
64 workers for (1) and 20 workers for (2) saturates instance with 64GB of RAM and 15K IOPS. Again, as the DB storage grows, a larger number of workers and faster storage should be considered.
```

File: content/docs/run-api-server/ingestion.mdx [hunk 3]
```diff
@@ -113,15 +113,15 @@ The following instructions assume the reingestion is done on AWS. However, they
...
 
   2. [Horizon](./installing.mdx) version 1.6.0 or newer (ideally, 2.x) installed on the machine from (1).
 
   3. [Core](https://github.com/stellar/stellar-core) version 17 or newer  installed on the machine from (1), given the network is on Protocol 17.
 
-  4. A Horizon database where to reingest the history. Preferably, the database should be at least an RDS `r4.8xlarge` instance or better (to take full advantage of its IOPS write capacity) and should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed). When using an RDS instance with General Purpose SSD storage, the reingestion throughput of the DB (namely Write IOPS) is determined by the storage size (3 IOPS per GB). With 5TB you get 15K IOPS, which can be saturated with 24 Horizon workers.
+  4. A Horizon database where to reingest the history. Preferably, the database should have at least 32 CPUs, 256 GB of RAM and at least 15K IOPS (forexample AWS RDS `r5.8xlarge`) and should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed).
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:122]:
```suggestion
  4. A Horizon database where to reingest the history. Preferably, the database should have at least 32 CPUs, 256 GB of RAM and at least 15K IOPS (for example AWS RDS `r5.8xlarge`) and should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed).
```

File: content/docs/run-api-server/ingestion.mdx [hunk 4]
```diff
@@ -162,15 +162,15 @@ export RETRY_BACKOFF_SECONDS=20
...
 
 (Where you can find `<latest_ledger>` under [SDF Horizon's](https://horizon.stellar.org/) `core_latest_ledger` field.)
 
-When saturating an RDS instance with 15K IOPS capacity:
+When saturating a database instance with 15K IOPS capacity:
 
 (1) should take a few hours to complete.
 
-(2) should take about 1.5 days to complete.
+(2) should take about 3 days to complete.
```
Comment (Shaptic) [inline content/docs/run-api-server/ingestion.mdx:173]: This is an unfortunate change :disappointed:

**Review by Shaptic — APPROVED** [review]
Thanks for this! Keeping our advice provider-agnostic is a good idea :+1:

File: content/docs/run-api-server/ingestion.mdx (hunk 4 above)
Comment (jacekn) [reply content/docs/run-api-server/ingestion.mdx:173]: Yes definitely! We have significantly more ledgers now and the network had been very busy in recent months which I'm sure contributed to this as well.

**Pull request merged**